config/crds/migration_v1alpha1_migmigration.yaml

```diff
@@ -30,6 +30,8 @@ spec:
           properties:
             migPlanRef:
               type: object
+            quiescePods:
```

Comment:
I don't know why `make manifest` changed this.

Reply:
It must have been missed like you say.